src/cogent3/core/sequence.py

```diff
@@ -1090,6 +1092,111 @@ def annotate_matches_to(self, pattern, annot_type, name, allow_multiple=False):
 
         return annot
 
+    def _repr_html_(self):
+        html = self.to_html(limit=self._repr_policy["num_pos"],)
```

Comment:
no quite right. Needs to truncate the sequence, as per the `__repr__` method. For example
```python
from cogent3 import make_seq

seq = make_seq("ACGGTGGGGGGGGG", moltype="dna")
seq
```
Displays `DnaSequence(ACGGTGG... 14)`. What I'm asking is the `ACGGTGG` be coloured. But see below.